src/components/navigation-bar/navigation-bar.pw.tsx

```diff
@@ -0,0 +1,313 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import Box from "../box";
+import NavigationBar, { NavigationBarProps } from ".";
+import NavigationBarComponent from "./components.test-pw";
+import { Menu, MenuItem } from "../menu";
+import {
+  Default,
+  DarkTheme,
+  WhiteTheme,
+  BlackTheme,
+  ExampleWithMenu,
+  IsLoading,
+  WithCustomSpacing,
+  ContentMaxWidthBox,
+  Sticky,
+  Fixed,
+} from "../../../src/components/navigation-bar/navigation-bar.stories";
```

Comment:
**question(non-blocking):** I thought if we were using stories/examples for Playwright tests, they had to be in the `./components.test-pw` file? I could well be wrong here (and tell me if I am because it'll save me a load of time moving the examples to a different file) but that's what I've been doing 😆